tests/azure/steps-run-tests.yml

```diff
@@ -47,6 +47,15 @@ steps:
         ./build/simple_test_from_il_binary_static
       displayName: Offline IL compilation simple tests
       condition: ${{ parameters.compilerAvailable }}
+    - script: |
+        set -ex
+        ./build/clspv -o simple.bc --emit-binary-ir ./tests/simple-from-llvm-ir-binary/simple.cl
+        export VK_ICD_FILENAMES=${{ parameters.icd_json_unix }}
+        export CLVK_LOG=3
+        ./build/simple_test_from_llvm_ir_binary
+        ./build/simple_test_from_llvm_ir_binary_static
+      displayName: Offline IR compilation simple tests
+      condition: ${{ parameters.compilerAvailable }}
```

Comment:
Wrong CI files :). We've switched to Github Actions months ago but I hadn't removed the Azure Pipelines files. I'm doing this with https://github.com/kpet/clvk/pull/431. Sorry for the confusion.